wgpu-hal/src/vulkan/device.rs

```diff
@@ -1260,11 +1260,12 @@ impl crate::Device for super::Device {
         desc: &crate::TextureViewDescriptor,
     ) -> Result<super::TextureView, crate::DeviceError> {
         let subresource_range = conv::map_subresource_range(&desc.range, texture.format);
+        let raw_format = self.shared.private_caps.map_texture_format(desc.format);
```

Comment:
note: It took me a bit to understand this change with keeping a `raw_format`, but for myself: Computing the raw format with `private_caps.map_texture_format` already happened once, and we're just trading some space for avoiding the lookup again later.